content/2016.12.give.a.new.chance.to.nightly.rst

```diff
@@ -0,0 +1,223 @@
+-----------------------------------------------------------------------
+
+Firefox is transitionning from a complex and error prone way of
+writing add-ons to a standardized way of writing Web extensions, which
+are most likely to be compatible with Chrome and use well known Web
+technologies which makes them easier to develop.
+
+This is challenging for add-ons creators that already have a working
+add-on that needs to be rewritten.
+
+As a community we can help them. If they already have a Google Chrome
+add-on, it is most likely that they will be able to install it in
+Firefox directly. Plus, Firefox exposes more add-ons API
+than Chrome and we will create more in 2017 to help as much as
+possible port previous add-ons to the Web extensions standard.
+
+As a user we can help to test these new add-ons to keep a good
+experience for our users.
+
+Have a better developer experience to develop your Web apps
+-----------------------------------------------------------
+
+If you develop Web applications you might have had some problems to
+debug them with Firefox.
+
+We have a full team of developers dedicated to make Firefox the best
+tool for developping Web applications. This team is improving Web
+developer tools everyday and by using Nightly it will make your life
+easier.
+
+I wish that at some point you would even want to open Firefox because
+it is easiser than debugging your app in Chrome. And I have good hope
+that we reach this point in 2017.
+
+
+Give a really valuable contribution to the Mozilla project without coding
+-------------------------------------------------------------------------
+
+Often people ask me how they can contribute to the Mozilla Firefox
+project.
+
+To be honest, starting to use Nightly today is a real huge step toward
+that goal.
+
+To use Firefox Nightly as a daily user is a simple and awesome way to
+help Firefox because it gives us feedback on the work we are doing
+ahead of time and also everyday. We don't have to wait for 18 weeks to
+get feedback on our work, we can have it the day after.
+
+By the time I write this article we have only few handfuls of users
+for Firefox Nightly so your choice to use it makes a huge difference
+to us.
+
+
+Conclusion
+----------
+
+I hope you are now convinced that you can help us to make a better
+Firefox by starting to use Firefox Nightly today.
+
+You can install the `desktop Firefox version <https://nightly.mozilla.org/>`_ as well as your
+`Android Firefox version <https://www.mozilla.org/firefox/channel/android/>`_
+
+By the time you will be using Firefox Nightly, **if you see something, say something**.
+
+You will have great chances to see it fixed the day after, while if
+you don't say anything it might never get fixed...
+
+`File a Bug in our bugtracker <https://bugzilla.mozilla.org/>`_ (you
+can now login with your Github account).
+
+Also in case of crashes, the bug report is sent automatically for you
```

Comment:
I think you could use `However` instead of `Also`, especially if you add something like, `so you don't need to do anything` to the paragraph somewhere.